tests/replication/test_federation_ack.py

```diff
@@ -25,8 +25,9 @@
 class FederationAckTestCase(HomeserverTestCase):
     def default_config(self) -> dict:
         config = super().default_config()
-        config["worker_app"] = "synapse.app.federation_sender"
-        config["send_federation"] = False
+        config["worker_app"] = "synapse.app.generic_worker"
+        # Workers with no name default to whatever is in 'worker_app'
+        config["federation_sender_instances"] = ["synapse.app.generic_worker"]
```

Comment:
Instead of explaining this, I think we should just be explicit:

```suggestion
        config["worker_app"] = "synapse.app.generic_worker"
        config["worker_name"] = "federation_sender1"
        config["federation_sender_instances"] = ["federation_sender1"]
```

Reply:
I like it. Done.